README.md

```diff
@@ -93,8 +93,7 @@ Components of the UI can be overridden by creating a `overrides.json` file. Ther
 
 - **name** - The name that is displayed in the plugin title bar. The default is "Regional Planning"
 - **description** - The text that is displayed when users hover on the plugin button. The default is "Configure and control layers to be overlayed on the base map".
-- **height** - The initial height of the plugin in pixels. The default is 400.
+- **size** - The size of the plugin: `'small'`, `'large'`, or `'custom'`. The default is `'small'`. If `'custom'` is selected the plugin's width will be set from the `'width'` property.
 - **width** - The initial width of the plugin in pixels. The default is 300.
```

Comment:
Consider updating the `width` documentation to note that it is required if `size` == `custom`. Also, I think we can drop `initial`. Since plugins are no longer resizable, it will always be this width.